### DISCUSSION TITLE
Any plan to support dragging in preview?

### DISCUSSION THREAD

**Discussion by weisszucker** [noah-nuebling/mac-mouse-fix#486] [Questions] [answered]
Finally I can zoom in with the wheel, what left is to drag the image with the mouse after zoom in. Is there any plan to support this functionality in the future?

**Comment by dmkallan** [comment]
This is very similar to what I was looking for.  My mouse has a clickable scroll wheel (middle button3).  The wheel can scroll while clicked.  I just want to use it to simulate the trackpad swipe-to-scroll and pinch-to-zoom gestures.  My ideal configuration is:

Click: Back
Long Click: Forward
Double Click: none (treat as two Clicks)
Click and Drag: Scroll (similar to two-finger swipe-to-scroll trackpad gesture)
Click and Scroll Wheel: Zoom (similar to pinch-to-zoom trackpad gesture, or how this app currently uses the Command key)

Any hope of getting something like this into a future version?

**Reply by noah-nuebling** [reply]
Yes! All of this is possible in the latest [3.0.0 Beta 6](https://github.com/noah-nuebling/mac-mouse-fix/releases/3.0.0-Beta-6).

**Comment by noah-nuebling** [comment, marked as answer]
Check out the latest [3.0.0 Beta 6](https://github.com/noah-nuebling/mac-mouse-fix/releases/3.0.0-Beta-6) ;)
